### PR TITLE
No more gradlew

### PR DESCRIPTION
**Pull request by dnfield**
Removes gradle wrapper from Android projects, updates build rules/scripts. This eliminates downloading gradle itself at build time, but does not eliminate other network connections gradle makes to download app dependencies.

I created the cipd package by downloading the gradle distribution, unzipping it,  and running:

```
cipd create -in gradle-7.0.2 -install-mode copy -tag version:7.0.2 -name flutter/gradle 
```

Part of flutter/flutter#88717

### CODE REVIEW

**Comment by zanderso** [comment]
The license checker has flagged an issue

**Comment by dnfield** [comment]
Looking into that - it's failing to realize this is apache licensed somehow.

**Comment by dnfield** [comment]
Ok, telling the tool to ignore `//third_party/gradle` since we only use it for build. This will likely trigger a different failure but it's tends to work better IME to get the new license tool hash from CI than to try to get it locally.

**Comment by dnfield** [comment]
Builds should succeed now.

File: testing/rules/run_gradle.py [hunk 1]
```diff
@@ -14,6 +14,11 @@
 import platform
 
 SCRIPT_PATH = os.path.dirname(os.path.realpath(__file__))
+
+BAT = '.bat' if sys.platform.startswith(('cygwin', 'win')) else ''
```
Comment (blasten) [inline testing/rules/run_gradle.py:18]: nit: BAT -> EXT

**Review by blasten — APPROVED** [review]
LGTM

**Comment by zanderso** [comment]
Presubmit still looks sad about the license tool signature. Otherwise lgtm

**Review by zanderso — APPROVED** [review]

**Pull request merged**